.github/workflows/nuget-publish.yml

```diff
@@ -24,9 +24,9 @@ jobs:
         run: npm install @openapitools/openapi-generator-cli -g
       - name: Generate serverside api
         run: bash ./generate/nuget.sh
-      - name: Move to generated api project
-        run: cd nuget/src/OneClickDesktop.Api
       - name: Build and pack library to nuget
-      - run: dotnet pack
+        run: dotnet pack
+        working-directory: ./nuget/src/OneClickDesktop.Api
       - name: Push nuget to nuget.org
         run: dotnet nuget push "bin/Release/OctocatApp.*.nupkg" --api-key ${{ secrets.NUGET_TOKEN }} --source "github"
```

Comment:
To jest zerzniete z poradnika przez przypadek xd
Moglbys poprawic na dobra nazwe? @VenMolom

Reply:
I nie wiem xzy source nie jeat bledny. Zaraz doczytqm dokumentacje i ci powiem

Reply:
Source tez jest jakis glupi z tutoriala.
Ustaw https://api.nuget.org/v3/index.json. Wtedy pakiet poleci w swiat.

Reply:
@smogork poprawione